documentation/modules/ROOT/pages/connectors/mysql.adoc

```diff
@@ -2784,7 +2813,15 @@ Note that changes to a primary key are not supported and can cause incorrect res
 
 |[[mysql-property-incremental-snapshot-chunk-size]]<<mysql-property-incremental-snapshot-chunk-size, `+incremental.snapshot.chunk.size+`>>
 |`1024`
-| The number of rows fetched from the database for each incremental snapshot iteration.
+|The maximum size, in bytes, of each chunk that the connector queries and reads into memory during an incremental snapshot.
```

Comment:
This is not in bytes, it is really number of rows, same for other connectors

Reply:
Ah. Ok, thanks. When I saw the `1024` I incorrectly interpreted that. Thanks for catching the error.